src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let sum = 0;
```

Comment:
let's calculate the total sum of rent for 40 days once and then just subtract the discount in each case